modules/dop/services/issue/import.go

```diff
@@ -17,29 +17,115 @@ package issue
 import (
 	"encoding/json"
 	"fmt"
-	"net/http"
+	"time"
+
+	"github.com/sirupsen/logrus"
 
 	"github.com/erda-project/erda/apistructs"
 	"github.com/erda-project/erda/modules/dop/dao"
 	"github.com/erda-project/erda/modules/dop/services/issueproperty"
 	"github.com/erda-project/erda/pkg/database/dbengine"
 )
 
-func (svc *Issue) ImportExcel(req apistructs.IssueImportExcelRequest, r *http.Request, properties []apistructs.IssuePropertyIndex, ip *issueproperty.IssueProperty, member []apistructs.Member) (*apistructs.IssueImportExcelResponse, error) {
-	// 获取测试用例数据
-	f, _, err := r.FormFile("file")
+const (
+	fileUploadFrom     = "autotest-space"
+	fileValidityPeriod = 7 * time.Hour * 24
```

Comment:
make it configurable

Reply:
adjusted, remove this configuration, becasue it's decided by front now
i made export issue file validity-period configurable
![image](https://user-images.githubusercontent.com/30427474/150474562-62377cc2-50b5-4c28-b0b5-d438c01d5a9d.png)